frontend/src/store/connection/selectors.ts

```diff
@@ -0,0 +1,22 @@
+import { createSelector } from 'reselect';
+import { StoreState } from '../types';
+import { ConnectionState } from './types';
+
+// Connection
+
+export const getConnection = (store: StoreState): ConnectionState => store.connection;
+
+// Backend
+
+export const getbackendHeartBeat = createSelector(
```

Comment:
Let's take this opportunity to fix the capitalization in this selector name, to `getBackendHeartbeat`

Reply:
I guess I misinterpreted the previous to-do where you probably meant `B` in HeartBeat should not be capitalized. will change if we still need backendHeartbeat and need to add it back